problems/arraysWithLoops.js

```diff
@@ -6,7 +6,14 @@
  * @returns {Array} - ["I!", "am!", "a!", "happy!", "array!"]
  */
 
-function shoutForLoop() {}
+function shoutForLoop(Array) {
```

Comment:
I still want you to fix these capitalized variables so make that change throughout the file